index.html

```diff
@@ -14,6 +17,101 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="container">
+
+    <header>
+      <nav>
+        <a href="index.html">Home</a>
+        <a href="#">About</a>
+        <a href="#">Portfolio</a>
+        <a href="#">Contact</a>
+      </nav>
+    </header>
+    <section class="main-container">
+      <section class="main-content">
+        <img class="logo" src="images/chair-daytime-desk-840996 (1).jpg" alt="man at desk image">
+        <h1>Johnny Appleseed - Web Dev</h1>
+        <h4>Technology is my jam!</h4>
+        <p> Occasionly I have time to work on side projects, if you're interested, let me know by signing up on this
+          website.
+        </p>
+        <button class="hdr-button">Lets Team Up</button>
+      </section>
+    </section>
+
+    <section class="top-content">
+      <section class="top-text">
+        <h2>Listen Up...</h2>
+        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Architecto ipsa, accusantium aspernatur est error
+          harum, inventore sed nihil libero quae tempora obcaecati ea dicta modi, quasi labore deleniti numquam neque
+          cupiditate. Dolore quo, facilis illo temporibus dignissimos qui inventore aperiam sapiente iste, similique
+          tempora! Minima consectetur facere quam reprehenderit, voluptas ex cupiditate magnam possimus fuga
+          voluptatibus, est, eos non itaque dolorem porro. Delectus adipisci quibusdam soluta provident excepturi
+          deserunt numquam mollitia voluptas facere maxime quisquam, a quas, ducimus, omnis quasi.</p>
+      </section>
+      <section class="top-list">
+        <h3>Stuff I'm Good At:</h3>
+        <ul>
+          <li>Disc Golf</li>
+          <li>Coding up</li>
+          <li>Playing lots of video games</li>
+          <li>Using bootstrap grid</li>
+          <li>Taking long naps</li>
+        </ul>
+      </section>
+    </section>
+
+    <section class="bottom-content">
+      <h2>Work History</h2>
+      <table class="worktable">
+        <tr>
+          <th>Yrs</th>
+          <th>Company</th>
+          <th>Title</th>
+          <th>Tech Used</th>
+        </tr>
+
+        <tr>
+          <td>1</td>
+          <td>Apple</td>
+          <td>Software Engineer</td>
+          <td>Swift</td>
+        </tr>
+
+        <tr>
+          <td>2</td>
+          <td>Microsoft</td>
+          <td>Computer Scientist</td>
+          <td>.NET</td>
+        </tr>
+
+        <tr>
+          <td>3</td>
+          <td>Amazon</td>
+          <td>Dev Ops</td>
+          <td>AWS</td>
+        </tr>
+      </table>
+    </section>
+
+    <section class="contact">
+
+      <h4>Contact</h4>
+      <address>
+        1600 Amphitheatre Parkway <br> Mountain View, CA 94043, USA
+        <br>
+        <br>
+        1 (541) 412-5695
+        <br>
+        <br>
+      </address>
+      Johnny@johnnyappleseed.com
```

Comment:
You could add email in the `<address>` tag too! If you wanted, you could even wrap those lines in paragraph tags instead of using `<br>` tags.

And specifically for the email and phone number, you could use [tel](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#Creating_a_phone_link) and [mailto](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#Creating_an_email_link) links here to let users easily open an email or text message to you.